website/common/locales/en/settings.json

```diff
@@ -210,5 +210,6 @@
     "mentioning": "Mentioning",
     "suggestMyUsername": "Suggest my username",
     "everywhere": "Everywhere",
-    "onlyPrivateSpaces": "Only in private spaces"
+    "onlyPrivateSpaces": "Only in private spaces",
+    "blurbIssueSlur": "About description may not contain inappropriate language nerd"
```

Comment:
This isn't at all a full review (one will be done later) but the final word in this string looks like it might be a typo. :)

I don't recall if the issue contained specific text to use. If it does, use that, and if it doesn't, can you change the string to the one below? Note the full stop at the end as well as the text changes.
"The About text may not contain inappropriate language."

Can you please also edit the top post on the PR to insert the issue number (that will create an automatic link to the issue) and to add your User ID? See the comments for where they should go. Your User ID is visible at [Settings > API](https://habitica.com/user/settings/api) on the website.